adafruit_magtag/magtag.py

```diff
@@ -327,7 +328,12 @@ def refresh(self):
         """
         Refresh the display
         """
-        self.graphics.display.refresh()
+        while True:
+            try:
+                self.graphics.display.refresh()
```

Comment:
Instead do `time.sleep(self.graphics.display.time_to_refresh)`.

Reply:
does `time_to_refresh` count down or is it the 'max seconds' to refresh